styles/globals.css

```diff
@@ -1,16 +1,36 @@
-@import url("https://fonts.googleapis.com/css2?family=Work+Sans:wght@100;200;300;400;500;600;700&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@200;300;400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap');
+
+@import url('./footer.css');
+@import url('./button.css');
 
 html,
 body {
   padding: 0;
   margin: 0;
-  font-family: "Work Sans", sans-serif;
+  font-family: 'Inter', sans-serif;
 }
 
 :root {
-  --primary-color: #7c3aed;
```

Comment:
Please make sure to remove `--primary-color` wherever it appears.

Reply:
@prerana1821 I've replaced all instances of `--primary-color` with `--primary`.